README.rst

```diff
@@ -26,6 +26,70 @@ current context is relevant.
 If the nearest sub site is not excluded but a parent is excluded, the message
 is shown on the current context.
 
+Allow automatic activation
+--------------------------
+
+With the ``Allow automatic activation?`` option active it is possible
```

Comment:
I removed the dummy class for the `IStatusMessageAutomaticEnable` and replaced `getUtility` with `queryUtility`.

Reply:
I would say to keep the `Allow automatic activation?` option for two reasons:

* having the option visible it's more explicit 
* it will cover the case when even if you have an automatic utility defined you may want to deactivate it and manually display the message